packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -136,6 +144,23 @@ const Tabs = ({
     }
   }, [value]);
 
+  // reset the scroll buttons override when the document becomes visible
+  const refreshScrollButtons = () => {
+    requestAnimationFrame(() => {
+      if (document.visibilityState === "visible") {
+        setScrollButtons("auto");
+      }
+    });
+  };
+
+  // listen for visibility change to reset scroll buttons override
+  useEffect(() => {
+    document.addEventListener("visibilitychange", refreshScrollButtons);
+    return () => {
+      document.removeEventListener("visibilitychange", refreshScrollButtons);
+    };
+  }, []);
+
```

Comment:
If we don't create this function in the `useEffect`, it will be hard to add cancelation.

In that case, we'll also need to use `let` unless we create a ref, and I think it's best to use `let` to encapsulate everything in the `useEffect` for this purpose.

Also, we need to check the `event` variable in the `visibilitychange` callback to make sure it's in the "active" state right? If it's already visible at the point of page load, I don't think we even need to run this code.

Lastly, we need to make sure we disable it completely once it's been set back to `auto` by putting a condition at the top of the `useEffect`. We can use `document.hidden`, I think, I figure that out at runtime along with the listener.